docs/src/upgrade/release-3.2.x-incubating.asciidoc

```diff
@@ -112,18 +112,19 @@ Gremlin Javascript can now submit script, with optional bindings, using the `Cli
 [source,javascript]
 ----
 const gremlin = require('gremlin');
-const connection = new gremlin.driver.Client('ws://localhost:8182/gremlin', { traversalSource: 'g' });
+const client = new gremlin.driver.Client('ws://localhost:8182/gremlin', { traversalSource: 'g' });
 
-connection.submit('g.V().tail()')
-    .then(response => {
-        console.log(response.traversers.length);
-        console.log(response.traversers[0]);
+client.submit('g.V().tail()')
```

Comment:
we usually don't edit past upgrade notes - they are static and bound to that version, but i think i see what you're trying to do here, i.e. hide away `traversers`....i guess that's ok

Reply:
Yes, the idea is to hide a deprecated property :)